docs/victory-axis/docs.md

```diff
@@ -206,6 +213,9 @@ The `events` prop takes an array of event objects. Event objects are composed of
             target: "tickLabels",
             mutation: () => {
               return {text: "hey"};
+            },
```

Comment:
_Sorry, I can't comment on the actual line because it's not part of this PR..._
Line 195 has a typo: `elemnts`